templates/main.php

```diff
@@ -29,8 +29,8 @@
                             <span class="lot__amount">Стартовая цена</span>
                             <span class="lot__cost"><?=priceModify($val['price']); ?></span>
                         </div>
-                        <div class="lot__timer timer">
-                            12:23
+                        <div class="lot__timer timer <?=$secDiff = secDifference($val['date_end']); oneHourTimerFinishing($secDiff); ?>">
```

Comment:
```suggestion
                        <div class="lot__timer timer <?= secDifference($val['date_end']); oneHourTimerFinishing($secDiff); ?>">
```
Присвоение в переменную тут лишнее. Убери плз пли следующем задании.